setup.py

```diff
@@ -29,7 +29,8 @@ def has_environment_marker_platform_impl_support():
     'w3lib>=1.19.0',
     'lxml',
     'six>=1.6.0',
-    'cssselect>=0.9'
+    'cssselect>=0.9',
+    'jmespath>=0.9',
```

Comment:
how did you decided this `jmespath` version?

Reply:
I have no idea, that is a really good question :slightly_smiling_face:

Reply:
I see no reason in https://github.com/jmespath/jmespath.py/blob/develop/CHANGELOG.rst to set a minimum version requirement.